templates/scenarios/ts/non-sso-tab/teamsapp.local.yml.tpl

```
@@ -31,6 +31,12 @@ provision:
   - uses: teamsApp/update # Apply the Teams app manifest to an existing Teams app in Teams Developer Portal. Will use the app id in manifest file to determine which Teams app to update.
     with:
       appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
+  - uses: m365Title/acquire # Upload your app to Outlook and the Microsoft 365 app
```

```suggestion
  - uses: teamsApp/extendToM365 # Extend your Teams app to Outlook and the Microsoft 365 app
```

Update to the above after prerelease merged back to dev.

---

Thanks for the information!